src/reducers/reducer_routes.js

```diff
@@ -0,0 +1,20 @@
+import { ActionConst } from 'react-native-router-flux';
```

Comment:
Unless it is a common convention for `react-native-router-flux`. I think we should rename this file to `reducer-routes.js` instead of `reducer_routes.js` or i think `routes.js`  because it is the the `reducers` module.